rtmp-client/src/main/cpp/flvmuxer/xiecc_rtmp.c

```diff
@@ -656,7 +667,11 @@ int rtmp_sender_write_video_frame(uint8_t *data,
             if (g_file_handle) {
                 fwrite(output, output_len, 1, g_file_handle);
             }
-            val += RTMP_Write(rtmp, output, output_len);
+            int result = RTMP_Write(rtmp, output, output_len);
```

Comment:
Does the `free(output)` need moving up to here?

Reply:
I've moved it.